### DISCUSSION TITLE
Add persian (Farsi) language on docs section

### DISCUSSION THREAD

**Discussion by alighdev** [kubernetes/website#45741] [Ideas]
Hi everyone and admin,
I have a question about translation doc sections. can I translate doc to persian and k8s team merged my pages to the main site?

thanks in advance.

**Comment by a-mccarthy** [comment]
Hi @alighdev! I'm Abbie one of the localization subproject leads along with @seokho-son. Thanks for your interest in localizing to Farsi. We have some documentation on our [localization processes](https://kubernetes.io/docs/contribute/localization/) if you are interested in learning more about localizing k8s documentation. Farsi is not currently a supported language so we don't allow folks to localize single documents at a time, but it is possible to add Farsi as a supported language by [starting a new localization](https://kubernetes.io/docs/contribute/localization/#start-a-new-localization). 

The best way to reach out with your questions are through slack in our [#sig-docs-localizations](https://kubernetes.slack.com/archives/C0191RDKHU1), we'd be happy to help asnwer your questions on localization processes there. We also have had some interest in localizing Farsi already. You can reach out in the Farsi language slack channel, [#kubernetes-docs-fa](https://kubernetes.slack.com/archives/C018HE5MA00), to find folks who are also interested in creating Farsi documentation. If you aren't a part of the k8s slack yet, you can get the invite: https://communityinviter.com/apps/kubernetes/community.

For now, i'll close this discussion, as they are usually used more for broader community discussions and not for questions. Looking forward to chatting more!

**Reply by alighdev** [reply]
Thanks @a-mccarthy for your good information that help me to translation.
